app/controllers/members_controller.rb

```diff
@@ -0,0 +1,76 @@
+class MembersController < ApplicationController
+	before_action :require_login
+
+	def new
+		@member = Member.new
+	end
+
+	def create
+		p params
+		@member = Member.new(member_params)
+		@member.family_id = current_user.id
+		@member.save
+		@member.assign_color
+		@member.save!
+		redirect_to :back
+	end
+
+	def destroy
+		if current_user == member.family
+			member = Member.find_by(id: params[:id])
+			if member.destroy
+				all_members = Member.all
+				render json: all_members
+			end
+		else
+			#can't delete someone not in your family.
+		end
+	end
+
+	def give_reward
+		p params
```

Comment:
Think about moving these reward methods into the models. They don't feel controller-y